src/Accelerators/NNPA/Conversion/ZHighToZLow/ZHighToZLow.cpp

```diff
@@ -32,15 +32,19 @@
 #include "src/Support/TypeUtilities.hpp"
 
 #define DEBUG_TYPE "zhigh-to-zlow"
-
+#define ENABLE_CSU_PAR true /* Allow parallel compiler gen Stick/Unstick. */
+#define CS_N 2              /* Tiling for Stick */
+#define CS_M 2              /* Tiling for Stick */
+#define CU_N 2              /* Tiling for Unstick */
+#define CU_M 2              /* Tiling for Unstick */
```

Comment:
Perhaps, moving them inside onnx_mIir::zhigh namespace is better.

Reply:
Agreed, once I am done with parameter tuning, I will migrate the files back down in the code, its easier to edit with vi all the terms in one spot.

Cleanup will also remove the alternative algorithms. Hiroshi also told me how to use big pages, which is what they did, possibly to avoid the prefetch issues. Time will tell if prefetch can alleviate the issues with dynamic shapes (which I think results in repetitive mallocs, which may exhibit less locality as what onnx-mlir does when sizes are static). This theory should be investigated, or maybe someone already knows :-)